test-utils/src/commonMain/kotlin/utils/AwaitUtils.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ Copyright 2020 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.test.utils
+
+import kotlinx.coroutines.delay
+
+val DefaultTimeout = 2000
```

Comment:
const?

Wouldn't it be convenient to group all constants related to timeout and delay values in a separate test-utils class or in the companion of a base test class?
I don't see how they should be different by test or by platform, but maybe you have some scenarios in mind